hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -352,12 +352,10 @@ private function up1402_removedUnusedManagedEntities($caseTypes, $activityTypes)
    * Removes unused ( or unneeded ) case types
    */
   private function up1402_removeUnusedCaseTypes() {
-    // Remove (Appraisal) & (Probation) case types
-    $caseTypesToRemove = ['Appraisal', 'Probation'];
+    // Remove (Probation) case types
+    $caseType = 'Probation';
 
-    foreach($caseTypesToRemove as $caseType) {
-      $this->removeUnusedCaseType($caseType);
-    }
+    $this->removeUnusedCaseType($caseType);
```

Comment:
You don't really need the variable here anymore, right? You can just `$this->removeUnusedCaseType('Probation')`